packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -60,14 +60,14 @@ void main() {
       await tester.pumpAndSettle();
       renderSelectionSpy.events.clear();
 
-      await gesture.moveTo(const Offset(200.0, 100.0));
+      await gesture.moveTo(const Offset(100.0, 200.0));
```

Comment:
why this change?

Reply:
This was because I was using `TapAndHorizontalDragGestureRecognizer`, but I have since changed it to `TapAndPan` so this change is reverted.